docs/user/exams/instructors_guide.rst

```diff
@@ -506,6 +510,20 @@ You can specify the moment when Artemis publishes the results of the exam, see `
 
    Exam Scores Page
 
+4.2. Exam Solutions
+^^^^^^^^^^^^^^^^^^^
+Optionally, you can publish the example solutions of text, modeling, file upload and programming exercises to students with submissions after a desired date by setting ``Example Solution Publication Date`` of the exam to a non-empty date.
+All exercises in the exam with the given types are published according to this date, as opposed to the course exercises which have their own individual example solution publication dates.
```

Comment:
Aren't the example solutions published?
And what types are you referring to?

Reply:
With `type`, I was referring to text, modeling, file upload and programming exercises. I can change the wording if it is not clear, do you have any suggestions?

Reply:
```suggestion
All example solutions of these exercises types are published according to this date set in the exam, as opposed to the course exercises which have their own individual example solution publication dates.
```